CHANGES.md

```diff
@@ -5,6 +5,8 @@ unreleased
 
 - Fix quoter and optimize identifier quoting (#327, @sim642)
 
+- Embed ppxlib’s error instead of raising, to keep the AST (#329, @panglesd)
```

Comment:
I'd say there are two levels on which this PR has impact on the user: one is that the ppx driver now returns a meaningful AST also when there's a ppxlib internal error. And the other one is that lots of functions in the ppxlib library now have a version returning a result instead of raising. Do you agree? If so, maybe you could even point that out by adding two different points to the changelog entry? Also, the one entry you currently have is about the driver part (the first point I've mentioned), so it might also be nice to make that a bit clearer; e.g. something along the lines
```suggestion
- Driver: In the case of ppxlib internal errors, embed those errors instead of raising to return a meaningful AST (#329, @panglesd)
```